code42_search_alerts.html

```diff
@@ -0,0 +1,129 @@
+{% extends 'widgets/widget_template.html' %}
+{% load custom_template %}
+
+{% block custom_title_prop %}{% if title_logo %}style="background-size: auto 60%; background-position: 50%; background-repeat: no-repeat; background-image: url('/app_resource/{{ title_logo }}');"{% endif %}{% endblock %}
+{% block title1 %}{{ title1 }}{% endblock %}
+{% block title2 %}{{ title2 }}{% endblock %}
+{% block custom_tools %}
+{% endblock %}
+
+{% block widget_content %} <!-- Main Start Block -->
+
+<!-- File: code42_search_alerts.html
+  Copyright (c) 2018-2021 Splunk Inc.
+  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+-->
+<style>
+
+    .code42v2 a:hover {
+        text-decoration:underline;
+    }
+
+    .code42v2 .wf-h4-style {
+        font-size : 20px;
+    }
+
+    .code42v2 .wf-table-vertical td{
+        padding: 5px;
+        border: 1px solid !important;
+    }
+
+    .code42v2 .wf-table-vertical tr td:first-child {
+        font-weight: bold;
+    }
+
+    .code42v2 .dataTable th.widget-th {
+        border-top: 1px solid !important;
+        border-bottom: 1px solid !important;
+    }
+
+    .code42v2 .dataTable td {
+        vertical-align: top;
+    }
+
+    .code42v2 .dataTable td.widget-td a{
+        font-size: 12px;
+    }
+
+    .code42v2 .dataTable {
+        font-size: 12px;
+    }
+
+</style>
+
+<div class="code42v2" style="overflow: auto; width: 100%; height: 100%; padding-left:10px; padding-right:10px">
+    <!-- Main Div -->
+
+    {% for result in results %} <!-- loop for each result -->
+        {% if not result.data %}
+            <h4 class="wf-h4-style">No alerts found</h4>
+        {% else %}
+
+            <div class="alert">
+                &nbsp;
+                <h4 class="wf-h4-style">Alerts</h4>
+                    <table class="phantom-table dataTable">
+                        <thead>
+                            <th class="widget-th">Actor</th>
+                            <th class="widget-th">State</th>
+                            <th class="widget-th">Severity</th>
+                            <th class="widget-th">Name</th>
+                            <th class="widget-th">Created At</th>
+                            <th class="widget-th">Description</th>
+                        </thead>
+                        <tbody>
```

Comment:
Allows for which widgets are clickable with contains-based actions and which are just text.

The view that opens up will also give `Indicator` information, which is fascinating. Like if the same user shows up in multiple alerts, etc.